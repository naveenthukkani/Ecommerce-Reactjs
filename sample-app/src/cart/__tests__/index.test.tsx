import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {CartItems} from '../index';

describe("CartItems", ()=>{

    it('renders Cart Screen Correctly', () => {
        const component = shallow(<CartItems />);
        expect(toJson(component)).toMatchSnapshot();
    });

});