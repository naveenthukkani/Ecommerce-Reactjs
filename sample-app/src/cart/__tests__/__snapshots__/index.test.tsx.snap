// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItems renders Cart Screen Correctly 1`] = `
<div
  className="Main-container"
>
  <div
    className="Adress-container"
  >
    <Address />
  </div>
  <div
    className="Cart-container"
  >
    <div>
      <Cart />
    </div>
  </div>
</div>
`;
